### ISSUE TITLE
[ec2] Inbound gossip network access is allowed on all instances from the internet

### ISSUE THREAD

**Issue by gozer** [Nubisproject/nubis-stacks#97]
From https://mana.mozilla.org/wiki/display/SECURITY/Nubis#Nubis-#inboundgossipopen

Inbound gossip network access is allowed on all instances from the internet

The nubis ec2 template allows inbound traffic on Consul gossip ports from all IPs instead of from the security group of other gossip group members (likely to be the SharedServiceSecurityGroup). This means that if an EIP is applied to any host or a host is created in the public subnet, an attacker can join the gossip group from the internet and siphon off traffic destined for legitimate service instances which in turn could result in exposure of restricted data

```
https://github.com/Nubisproject/nubis-stacks/blob/cde5aa4ed0b2a35aff28ad194b1a03fe163dc899/ec2.template#L199-L210
```

Recommendations :

Change the security group to only allow an origin value of a security group that contains all the gossip members

**Comment by gozer** [comment]
This was resolved by simply removing these inbound rules, as they were already being inherited by the SharedServiceSecurityGroup, which is exactyl as described.

It only allows Consul connections from members of said security group

**Comment by gene1wood** [comment]
This was fixed with commit c5dfd460fecae189469f38c21cc7b8cc42160cb0
